eng/packages/http-client-csharp/generator/TestProjects/Local/Basic-TypeSpec/src/Generated/Internal/ErrorResult.cs

```diff
@@ -5,22 +5,26 @@
 
 #nullable disable
 
-using System.ClientModel;
-using System.ClientModel.Primitives;
+using Azure;
 
 namespace BasicTypeSpec
 {
-    internal partial class ErrorResult<T> : ClientResult<T>
```

Comment:
We should remove this from unbranded clients.  SCM-based clients should never throw from a property.